jwst/scripts/v1_calculate.py

```diff
@@ -121,3 +121,7 @@
     formatted = v1_calculate.simplify_table(v1s)
     formatted.write(args.output, format='ascii.ecsv')
     logger.info('...V1 calculation completed.')
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
I know this is merged already, but generally, if you are installing via console scripts entry points, you no longer need this in the script itself, especially if you are pointing to `main()` in the entry point.